mozilla-release/toolkit/mozapps/extensions/addonManager.js

```diff
@@ -174,6 +176,15 @@ amManager.prototype = {
         return AddonManager.isInstallEnabled(payload.mimetype);
 
       case MSG_INSTALL_ADDON: {
+      /* Cliqz start */
+      // prevent installing Ghostery
+      if(payload.hash ===
+      "sha256:2fcf1ba8a7ff7344db4c26e3f06765dac2250b328c61d45b36f2a79015128ad5"||
```

Comment:
uhm, a sha256. Are you sure this is not just the hash of the xpi? If so, this will work for the current release but not if we push a new AMO version